Axial_Descent_Ascent_Operating_Curves_Rotor.m

```diff
@@ -0,0 +1,244 @@
+    end
+    for i = 1 : length(hh)
+        plot3(hh(i)*ones(1,length(PTD(i,1:bb(i)))),VVd(1,1:bb(i)),PTD(i,1:bb(i)),'k')
+    end
+    yticks([-50 -25 0 25 50])
+    xlabel('$Height[m]$','Interpreter','latex', 'FontSize',15)
+    ylabel('$V_{\infty}[\frac{m}{s}]$','Interpreter','latex', 'FontSize',15)
+    zlabel('$\widetilde{P}$','Interpreter','latex', 'FontSize',15)
+    grid on
+    title('$P (V_{\infty}, h)$','Interpreter','latex', 'FontSize', 15)
+    view(71,32)
+    
+    %% Insertion of Interes Altitude;
+    
+    prompt = {'Insert interest altitude in metres [min=0,Max=6000]: '};
+    dlgtitle = 'Altitude';
+    dims = [1 35];
+    answer = inputdlg(prompt,dlgtitle,dims);
+    hnew = str2double(answer{1});
+    
+    WTSnew = zeros(1,length(VVs));  %Vector Inizialization of axial ascent induction related to velocity
+    WTDnew = zeros(1,length(VVd));  %Vector Inizialization of axial descent induction related to velocity
+    
+    % Matrices fill
+    for j = 1 : length(VVs)
+        WTSnew(1,j) = w_tilde_salita(VVs(j),hnew);
+    end
+    
+    for j = 1 : length(VVd)
+        if V_tilde(VVd(j),hnew) <= -2                  %Validity limit of simply impulsive theory;
+            WTDnew(1,j) = w_tilde_discesa(VVd(j),hnew);
+            aanew = j;
+        else
+            break
+        end
+    end
+    
+    PTSnew = zeros(1,length(VVs));
+    PTDnew = zeros(1,length(VVd));
+    
+    
+    for j = 1 : length(VVs)
+        PTSnew(1,j) =  P_tilde_salita(VVs(j),hnew);
+    end
+    
+    for j = 1 : length(VVd)
+        if V_tilde(VVd(j),hnew) <= -2                  %Validity limit of simply impulsive theory;
+            PTDnew(1,j) = P_tilde_discesa(VVd(j),hnew);
+            bbnew = j;
+        else
+            break
+        end
+    end
+    
+    %% 2D Plots of induction [Output]
+    figure(3)
+    plot(V_tilde(VVs,hnew), WTSnew, '-k')
+    hold on
+    plot(V_tilde(VVd(1:aanew),hnew), WTDnew(1:aanew), '-k')
+    grid on
+    xlabel('$\widetilde{V}$','Interpreter','latex','FontSize',15)
+    ylabel('$\widetilde{w}$','Interpreter','latex','FontSize',15)
+    title(['$\widetilde{w}$ per h=' num2str(hnew) '[m]'],'Interpreter','latex')
+    
+    % 2D Plots of power [Output]
+    figure(4)
+    plot(V_tilde(VVs,hnew), PTSnew, '-k')
+    hold on
+    plot(V_tilde(VVd(1:aanew),hnew), PTDnew(1:bbnew), '-k')
+    grid on
+    xlabel('$\widetilde{V}$','Interpreter','latex','FontSize',15)
+    ylabel('$\widetilde{P}$','Interpreter','latex','FontSize',15)
+    title(['$\widetilde{P}$ per h=' num2str(hnew) '[m]'],'Interpreter','latex')
+    %% Numerical Output of Power and Induction for the interest altitude;
+    Power = [PTDnew(1:bbnew), PTSnew];
+    Induction = [WTDnew(1:aanew), WTSnew];
+  
+elseif nargin==3 % In Output only value of Power and Induction
+                 % at altitude and velocity of interest;
+    prompt = {'Insert interest altitude in metres [min=0,Max=6000]: '};

```

Comment:
Do not ask the user to type a value.
Give it in input, or read it from the input text file when the interface functions will be ready.